android/app/build.gradle

```diff
@@ -95,6 +95,7 @@ project.ext.react = [
     entryFile: "index.js",
     bundleInDevDebug: false,
     bundleInStagingDebug: true,
```

Comment:
@efalkner I think we might be able to get rid of `bundleInStagingDebug`, as that's no longer used? double checking it's not used by BT

Reply:
@JacobJaffe @efalkner presumably `bundleInBteStaging` should be added too?